TRAVIS

```diff
@@ -9,34 +16,31 @@ Implement a track test suite that can run both locally and on Travis CI. The tra
 
 **Background**
 
-When implementing an exercise test suite, we want to provide a good user experience for the people writing a solution to the exercise. People should not be confused or overwhelmed.
-
-In most Exercism language tracks, we simulate Test-Driven Development (TDD) by implementing the tests in order of increasing complexity. We try to ensure that each test either
+The goal of the "track test suite" is to ensure that the "exercise test suite"s delivered to Exercism users will work properly.  The best way to do this is to execute the exercise test suite against a known good solution — the example solution in the exercise.
 
-- helps triangulate a solution to be more generic, or
-- requires new functionality incrementally.
+_Enabling all test cases_
 
-Many test frameworks will randomize the order of the tests when running them. This is an excellent practice, which helps ensure that subsequent tests are not dependent on side effects from earlier tests. However, in order to simulate TDD we want tests to run *in the order that they are defined*, and we want them to *fail fast*, that is to say, as soon as the test suite encounters a failure, we want the execution to stop. This ensures that the person implementing the solution sees only one error or failure message at a time, unless they make a change which causes prior tests to fail.
+As described in [Contributing to Exercism: Writing exercise test suites](writing_tests.md), exercise test suites are written such that all but the first test case are skipped/pending/ignored.  For the purpose of the track test suite, all test cases should be enabled (simulating an Exercism user having completed the exercise).
```

Comment:
- `s/Writing/writing/`
- `s/writing_tests.md/writing-tests.md`

> exercise test suites are written such that all but the first test case are skipped/pending/ignored.

`s/test suites/test suites in most language tracks/`. Some language tracks are able to implement this without skipping, pending, ignoring (e.g. Go has a `t.Fatal()` that will stop on the first error or failure, whereas `t.Error()` will report the error and move on. I think also some language tracks have decided not to fail fast. That might be a decision that we need to consider when we start looking at the curriculum from a high level and consider the user experience.

I really like the _simulating an Exercism user having completed the exercise_ bit.